api/src/gmsa_service.cpp

```diff
@@ -7,6 +7,8 @@
 #include <grpcpp/grpcpp.h>
 #include <grpcpp/health_check_service_interface.h>
 #include <random>
+#include <sys/stat.h>
```

Comment:
what is this reference used for?

Reply:
@saikiranakula-amzn I will check later today

Reply:
@saikiranakula-amzn This is required for use of S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH variables in the gmsa_service.cpp